app/services/application_association_backend_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ApplicationAssociationBackendService
+  def delete_all(application_id:, service_backend_id:, application_backend_id:)
+    DeletedObject.public_send(reflection_name).where(owner_type: Contract, owner_id: application_id).order(:id).find_each do |deleted_object|
```

Comment:
what's the default for `order(:id)` i would rather to see in here asc/desc.

if ower_type could be a string, `Contract` would be better.. using classes could end up with random errors.

Reply:
the default is :asc . But I do not really care about that as long as it keeps an order... but in postgres it would be 'random' otherwise.

> if ower_type could be a string, Contract would be better.. using classes could end up with random errors.

You are totally right! Will do! In fact, it is deprecated doing it this way and in future versions of Ruby (I think already in 5.1), it will always fail for sure! 😱 good catch!

Reply:
Done 😎